### DISCUSSION TITLE
Button View More is too small

### DISCUSSION THREAD

**Discussion by bernardocamps** [community/community#3678] [Discussions]
In the web version, the button "View More" is too small. 
It took me a while to realize that there was one more comment on the thread.

![Captura de Tela 2021-04-02 às 11 41 25](https://user-images.githubusercontent.com/62954018/113425520-5f719480-93a8-11eb-8914-e9c7d877b416.png)

The iOS version allows better viewing:

![Captura de Tela 2021-04-02 às 11 54 31](https://user-images.githubusercontent.com/62954018/113426769-541f6880-93aa-11eb-9099-3442cbc5a0e7.png)

**Comment by gilgongo** [comment]
It's also a bit mysterious as to why it's there in the the first place. It seems to be encouraging people not to read subsequent posts, or to create new sub-threads. I think I've seen people post on threads who have no seen the discussions under "view more" so you have duplicate discussions going on inside "view more" jails. Somewhat weird,
